Upstream release contains config.{guess,sub} from ten years ago that
don't handle Apple silicon. This patches minimally around it.

--- a/config.sub	2014-06-10 20:03:11
+++ b/config.sub	2024-07-23 19:48:49
@@ -436,6 +436,9 @@
 	| ymp-* \
 	| z8k-* | z80-*)
 		;;
+	arm64-apple)
+		basic_machine=aarch64-apple
+		;;
 	# Recognize the basic CPU types without company name, with glob match.
 	xtensa*)
 		basic_machine=$basic_machine-unknown
